autopts/wid/gatt_client.py

```diff
@@ -1289,6 +1290,49 @@ def hdl_wid_109(params: WIDParams):
     return True
 
 
+def hdl_wid_130(params: WIDParams):
+    """
+    Please delete security key before connecting to PTS if IUT was bonded previously.
+    """
+    btp.gap_unpair()
+
+    return True
+
+
+def hdl_wid_135(params: WIDParams):
+    """
+    Please write to client support feature handle = 'XXXX'O to enable Robust Caching.
+    Discover all characteristics if needed.
+    """
+
+    pattern = re.compile(r"'([0-9a-fA-F]+)'")
+    params = pattern.findall(params.description)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+
+    # First read the existing value in Client Supported Features.
+    btp.gatt_cl_read(btp.pts_addr_type_get(), btp.pts_addr_get(), handle)
+    stack = get_stack()
+    stack.gatt_cl.wait_for_verify_values()
+    client_support_features = int(btp.get_verify_values()[1])
+
+    # Set Robust Caching bit in features.
+    value = (
+        client_support_features
+        | defs.GATT_CLIENT_SUPPORTED_FEATURES_ROBUST_CACHING
+    )
```

Comment:
`get_verify_values` isn't documented, so I'm not sure what it does, but is there a reason why `GATT_CLIENT_SUPPORTED_FEATURES_ROBUST_CACHING` isn't added to that?

Reply:
Thanks :) 

I think I'm missing some overall design knowledge here, so I don't think I can approve this PR (simply because I don't know if it's correctly implemented) - So I defer to someone else to approve :)